client/utils/load-translation.js

```diff
@@ -0,0 +1,89 @@
+import {addLocale, useLocale} from "ttag";
+
+const setLocale = (language, translationObj) => {
+  addLocale(language, translationObj);
+  useLocale(language);
+};
+
+const loadCustomTranslation = (orgSlug, language) =>
+  import(`../translations/${orgSlug}_${language}.custom.json`);
+const loadDefaultTranslation = (language) =>
+  import(`../translations/${language}.json`);
+
+const importTranslation = async (orgSlug, lang) => {
+  try {
+    const {default: translationObj} = await loadCustomTranslation(
+      orgSlug,
+      lang,
+    );
+    setLocale(lang, translationObj);
+  } catch (err) {
+    const {default: translationObj} = await loadDefaultTranslation(lang);
+    setLocale(lang, translationObj);
+  }
+};
+
+const loadTranslationUtil = async (
+  language,
+  orgSlug,
+  useBrowserLang,
+  availableLanguages,
+) => {
+  let lang = language;
+  if (useBrowserLang) {
+    const availLangs = availableLanguages.map((langObj) => {
+      return langObj.slug;
+    });
+    navigator.languages.every((browserLang) => {
+      if (availLangs.includes(browserLang)) {
+        lang = browserLang;
+        return false;
+      }
+      if (availLangs.includes(browserLang.substr(0, 2))) {
+        lang = browserLang.substr(0, 2);
+        return false;
+      }
+      return true;
```

Comment:
does this `true` and do the above `false` values returned here make any difference?
It's a bit confusing.

Reply:
It's just to break the iteration.

Reply:
then you should be able to simply use `return`, the last 2 return statements should not be needed, or is the linter complaining? Alternatively just use `else if` insead of the second `if` in which case you should be able to avoid the return completely and no ambiguity is left.